towncrier.toml

```diff
@@ -65,3 +65,8 @@ showcontent = true
 directory = "internal"
 name = "Internal Changes"
 showcontent = true
+
+[[tool.towncrier.type]]
+directory = "performance"
+name = "Performance Improvements"
+showcontent = true
```

Comment:
👍 but the doc and issue template needs to be updated for this.

Reply:
No problem, but not sure where to look. Were you thinking of the PR template?

Reply:
Yes but nvm we changed it to this recently: https://github.com/pylint-dev/pylint/blob/main/.github/PULL_REQUEST_TEMPLATE.md?plain=1#L9